Easy/TwoIntegerSum/solution.js

```diff
@@ -0,0 +1,44 @@
+class Solution {
+    /**
+     * Solução
+     *  - Primeiro passo: Criei um hashmap para armazenar informações de cada elemento do array passado
+     *      valor = complemento(um valor que somado com o do elemento resulta no target).
+     *      chave = posição do elemento dentro do array nums.
+     * 
+     *  - Segundo passo: Criei um loop para pega um elemento do array nums, calcula seu complemento e adicionar
+     *      o valor e chave dentro do hashmap.
+     * 
+     *  - Terceiro passo: Fiz uma verificação dentro do loop, se seu complemento se encontra dentro do hashmap é retornado o elemento e
+     *      seu complemento.No entanto, caso seja encontrado o complemento é retornado um array vazio.
+     * 
+     *  Big O Espaço: O(n)
+     *  Big O Tempo: O(n)
+     * 
+     *  Tempo necessário para solução: 2h
+     * 
+     *  Precisei pesquisar sobre Map.has(): https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Map/has
+     * 
+     * 
+     * @param {number[]} nums = array de numeros passado.
+     * @param {number} target = valor da soma de 2 elementos dentro de nums. 
+     * @return {number[]} = sempre irá retorna um array de 2 elementos ou vazio.
```

Comment:
Não faz sentido atribuir o valor da descrição aqui 🤔. Não é recomendado colocar algo que relaciona o tipo da variavel na descrição. Tente sempre descrever a variável como o que ela é na perspectiva de um leitor que está vendo a documentação de um sistema. Enquanto o retorno tem que ter informações sobre o que é esperado sem duplicidade.

Minha recomendação para essa descrição seria algo como

```suggestion
     * @param {number[]} nums uma lista de numeros.
     * @param {number} target valor esperado.
     *
     * @return {number[]} uma lista de dois numeros que somado dá o valor esperado, ou uma lista vazia caso não encontrado.
```